javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,60 @@
+//SANDBOX URL: https://codesandbox.io/s/talk-to-your-computer-ph2ql?file=/src/index.js
+
+function getReply(question) {
+  switch (question) {
+    case 'Hello my name is Deepti':
+      const nameArray = question.split(' ');
+      if (question.startsWith('Hello my name is')) {
+        const name = nameArray[nameArray.length - 1];
+        return `Nice to meet you ${name}`;
+      }
```

Comment:
This is still not dynamic. I want you to get any name not only your name. If the command is `'Hello my name is John' for example. Then if the command is `What is my name?` You should return `John` You can use the array that you get from `split` or you can replace everything except the name with an empty string like
```js
const name = command.replace('Hello my name is ', '');
```
I recommend using `if` `else` instead of `switch`
Looking for updating this and the other command options similar way